QUICKSTART.md

```diff
@@ -0,0 +1,165 @@
+})
+export class AppModule {}
+```
+
+### 2: Meta Reducer
+
+Add the autoEntityMetaReducer to your meta reducers for your application or
+feature:
+
+```typescript
+import { routerReducer } from '@ngrx/router-store';
+import { ActionReducerMap, MetaReducer } from '@ngrx/store';
+import { autoEntityMetaReducer } from '@briebug/ngrx-auto-entity';
+import { storeFreeze } from 'ngrx-store-freeze';
+// ...
+import { environment } from '../../environments/environment';
+
+export interface IAppState {
+  // ...
+}
+
+export const appReducer: ActionReducerMap<IAppState> = {
+  // ...
+};
+
+export const appMetaReducers: Array<MetaReducer<IAppState>> = !environment.production
+  ? [autoEntityMetaReducer, storeFreeze]
+  : [autoEntityMetaReducer];
+```
+
+### 3: App State
+
+Add default `EntityEffects` to your effects in your state module and reference
+your meta reducers in the store:
+
+```typescript
+// ...
+import { EntityEffects } from '@briebug/ngrx-auto-entity';
+import { appMetaReducers, appReducer } from './app.reducer';
+// ...
+
+@NgModule({
+  imports: [
+    // ...
+    StoreModule.forRoot(appReducer, { metaReducers: appMetaReducers }),
+    EffectsModule.forRoot([EntityEffects /* ... */])
+    // ...
+  ]
+})
+export class StateModule {
+  // ...
+}
+```
+
+### 4: Entity State
+
+You will need to set up basic state for each entity. For fully automatic entities, you
+only need a basic reducer and standard selectors:
+
+```typescript
+import { Customer } from 'models';
+import { buildState, EntityActions, IEntityState } from '@briebug/ngrx-auto-entity';
+
+const { initialState, selectors } = buildState(Customer);
+
+export const {
+  selectAll: allCustomers,
+  selectEntities: customerEntities,
+  selectIds: customerIds,
+  selectTotal: totalCustomers
+} = selectors;
+
+export function customerReducer(state = initialState, action: EntityActions<Customer>): IEntityState<Customer> {
+  return state;
+}
+```
+
+And make sure you register the state and reducer in your main state code:
+
+```typescript
+import { routerReducer } from '@ngrx/router-store';
+import { ActionReducerMap, MetaReducer } from '@ngrx/store';
+import { autoEntityMetaReducer } from '@briebug/ngrx-auto-entity';
+import { storeFreeze } from 'ngrx-store-freeze';
+import { customerReducer } from 'state/customer/customer.reducer';
+import { environment } from 'environments/environment';
+
+export interface IAppState {
+  customer: IEntityState<Customer>;
+}
+
+export const appReducer: ActionReducerMap<IAppState> = {
+  customer: customerReducer
+};
+
+export const appMetaReducers: Array<MetaReducer<IAppState>> = !environment.production
+  ? [autoEntityMetaReducer, storeFreeze]
+  : [autoEntityMetaReducer];
+```
+
+### 5: Models & Services
+
+You are now ready to get rolling! You will need to create your models and your
+services, however for a simple application you are now fully set up. Just make sure
+you properly register your providers in your module:
```

Comment:
specify registering providers in the app.module

Reply:
Could alternatively be feature state module...